core/warning.js

```diff
@@ -150,6 +156,12 @@ Blockly.Warning.prototype.createBubble = function() {
     }
   }
   this.applyColour();
+
+  // Allow the bubble to be dragged from the warning paragraph
+  if (!this.block_.workspace.options.readOnly) {
+    Blockly.bindEventWithChecks_(
```

Comment:
This works, but there's really no point of the additional code. The backing layer (``bubbleBack_``) in bubble is handling this behaviour already. All you really need to do is not handle events on the paragraph layer. You can do this by setting ``pointer-events:none``.
I would suggest adding a css class ``blocklyNoPointerEvents`` in css.js that does: 
```
.blocklyNoPointerEvents {
  pointer-events: none;
}
```
and setting the class of the paragraph to that.